### DISCUSSION TITLE
Translating README into other languages?

### DISCUSSION THREAD

**Discussion by Ttiki** [ravynsoft/ravynos#238] [Q&A] [answered]
Hello everyone.

I would love to help support this project, but I never worked on a project like this (this big) before. However, I was asking myself if it would be interesting to translate the README into other languages. At the moment there is only Italian and English, but using a service like [crowdin](https://crowdin.com/) could help with newcomers from around the world.
I know it's not the biggest and most important part, but I was just questioning myself if you would be interested in this?

**Comment by comradekingu** [comment]
Happy to help if you use Weblate and remove the CoC.

**Reply by mszoek** [reply]
Why remove the Code of Conduct?

**Reply by comradekingu** [reply]
If not for my own benefit, I only have community and the values found in the license left.

**Comment by mszoek** [comment, marked as answer]
Hi @Ttiki! Thanks for the offer! You're welcome to translate the README or any parts of wiki.airyx.org for now - that would be really helpful. (Just submit it as a pull request or PM/email me your translation.)

Writing new docs/wiki content is also helpful. We're still too early to have much documentation and it's pretty fluid, so something like crowdin or weblate is probably not a good use of time yet. Definitely later, though.
